src/modules/home/components/Postings.js

```diff
@@ -18,7 +19,11 @@ export const Postings = () => {
 
     if (!posts || !posts.length) {
```

Comment:
We shoulkdnt be using this logic for loading. If our app had npo posts we would get a infinite spinner. 
Add in a useState for loading
const [isLoading, setIsLoading] = useState(false)

Usually a site would have a whole other page for empty states. eg Wow it looks quiet today. That type of shit you see.

Reply:
done